src/modules/Popup/Popup.js

```diff
@@ -195,7 +251,7 @@ export default class Popup extends Component {
         ) : (
           children
         )}
-        {hideOnScroll && <EventStack on={this.hideOnScroll} name='scroll' target='window' />}
+        {hideOnScroll && <EventStack on={hideOnScroll} name='scroll' target='window' />}
```

Comment:
@layershifter after refactoring the Popup to FC, there is a `hideOnScroll` callback and prop name clash. It causes the guard be always truthy and thus the prop does not work. The callback should be renamed.

I think that this PR should be reviewed again because there might be more occurrences of this same issue. What do you think?

Reply:
@dominikdosoudil good catch, please trigger an issue and feel free to submit a PR 👍

Reply:
Apparently there exists both issue and PR #4490 (at least for the popup)